components/devtools/actors/network_event.rs

```diff
@@ -370,7 +371,18 @@ impl NetworkEventActor {
             actor: self.name(),
             url: self.request.url.clone(),
             method: format!("{}", self.request.method),
-            startedDateTime: format!("{}", self.request.startedDateTime.rfc3339()),
+            startedDateTime: format!(
+                "{}",
+                (Local::now() -
+                    Duration::from_secs(
+                        self.request
+                            .startedDateTime
+                            .duration_since(UNIX_EPOCH)
+                            .unwrap_or_default()
+                            .as_secs()
+                    ))
+                .to_rfc3339()
```

Comment:
This doesn't seem like the right way to calculate the local time. You are creating a Local from the start time. You are creating a local time here and subtracting the timestamp of the start time. I think that you should look into `Local::timestamp_opt` in order to create the `Local`. In addition, I think that millisecond accuracy would be a better fit here.

Reply:
Indeed that was not right, thanks for proposing `timestamp_opt`, I used it.